.github/actions/conventional-commits/action.yml

```diff
@@ -6,7 +6,7 @@ runs:
     - uses: actions/github-script@v5
       with:
         script: |
-          const validator = /^(chore|feat|fix|revert|docs|style)(\([a-z ]+\))?(!)?: (.)+$/
+          const validator = /^(chore|feat|fix|revert|docs|style)(\((CCIE|CDI|ONCALL)-[0-9]+\))?(!)?: (.)+$/
```

Comment:
can you provide a https://regexr.com/ link showing this works as expected? I just tried it and I am not sure it captures exactly as expected.

Reply:
regexr.com/6r60u
The original ticket wants to enforce <type><JIRA>: <title> but currently it's <type>: <scope/JIRA> <title>
Open question that I noted just now is, what do we do about the automated PRs that aren't attached to a JIRA ticket?

Reply:
I am just asking for a link to a regex testing website showing this pattern captures what you expected because I am very bad at reading regex

Reply:
oops I think this might have gotten lost
https://regexr.com/6r60u

Reply:
awesome thanks!